src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6847,6 +6847,18 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_DoNotCompareFunctionPointers_Title" xml:space="preserve">
     <value>Do not compare function pointer values</value>
   </data>
+  <data name="IDS_FeatureUsingTypeAlias" xml:space="preserve">
+    <value>using type alias</value>
+  </data>
+  <data name="ERR_BadRefInUsingAlias" xml:space="preserve">
+    <value>Using alias cannot be a 'ref' type.</value>
+  </data>
+  <data name="ERR_BadUnsafeInUsingDirective" xml:space="preserve">
+    <value>Only a using alias can be 'unsafe'.</value>
+  </data>
+  <data name="ERR_BadNullableReferenceTypeInUsingAlias" xml:space="preserve">
+    <value>Using alias cannot be a nullable reference type..</value>
```

Comment:
Unnecessary second dot at the end of the message

Reply:
Fixed.